fusor-ember-cli/app/controllers/register-nodes.js

```diff
@@ -327,10 +327,9 @@ export default Ember.Controller.extend({
   updateAfterRegistration: function(resolve) {
     var self = this;
     var deploymentId = this.get('deploymentId');
-    self.get('model').nodes.store.find('node', {deployment_id: deploymentId, reload: true}).then(function() {
-      self.get('model').profiles.store.find('flavor', {deployment_id: deploymentId, reload: true}).then(function () {
-        if (resolve)
-        {
+    this.store.find('node', {deployment_id: deploymentId, reload: true}).then(function() {
```

Comment:
I confirmed in the console that `self.get('model').nodes.store` returns the same as `this.store`. There is just one store.